src/xml_file_loader.cc

```diff
@@ -339,7 +339,7 @@ void XMLFileLoader::LoadPackages() {
     std::string name = cyclus::OptionalQuery<std::string>(qe, "name", "default");
     CLOG(LEV_DEBUG3) << "loading package: " << name;
     
-    double fill_min = cyclus::OptionalQuery<double>(qe, "fill_min", 0);
+    double fill_min = cyclus::OptionalQuery<double>(qe, "fill_min", eps());
```

Comment:
What is this `eps()` method?

Reply:
This is the epsilon that either a user defines in their input file or a default is applied